activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -67,7 +67,7 @@ def sql_color(sql)
       case sql
         when /\A\s*rollback/mi
           RED
-        when /\s*.*?select .*for update/mi, /\A\s*lock/mi
+        when /select .*for update/mi, /\A\s*lock/mi
```

Comment:
I see what you mean. I don't have strong opinion on it, just don't want to slowdown the requests because a logging coloring.